src/ngx_http_lua_module.c

```diff
@@ -627,6 +641,11 @@ ngx_http_lua_init(ngx_conf_t *cf)
 {
     int                         multi_http_blocks;
     ngx_int_t                   rc;
+    ngx_uint_t                  i;
+    size_t                      len;
+    u_char                     *path;
+    u_char                     *tmp;
+    ngx_str_t                  *p;
```

Comment:
We usually use `p` for C string pointers only. Better use a different name for it.